chsdi/templates/htmlpopup/vogelreservate.mako

```diff
@@ -13,8 +13,8 @@
     <tr><td class="cell-left">${_('ch.bafu.bundesinventare-vogelreservate.teilgebiet')}</td>         <td>${c['attributes']['teilgebiet'] or '-'}</td></tr>
     <tr><td class="cell-left">${_('ch.bafu.bundesinventare-vogelreservate.schutzkategorie')}</td>         <td>${c['attributes'][schutzkat] or '-'}</td></tr>
     <tr><td class="cell-left">${_('ch.bafu.bundesinventare-vogelreservate.schutzebene')}</td>         <td>${c['attributes'][schutzeb] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.bundesinventare-vogelreservate.shape_area')}</td>          <td>${round(c['attributes']['shape_area']/10000) or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.bundesinventare-vogelreservate.obj_gisflaeche')}</td>         <td>${round(c['attributes']['obj_gisflaeche']) or '-'}</td></tr>
```

Comment:
I don't know the exact spec for this layer but would it not be better to round the data that is written in the DB so it will be the same every where and not just for the tool-tip?